### ISSUE TITLE
Right-click context item to copy/open URL

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup#246]
For accessibility purposes, the URL should be copy-able and open-able from the context menu.

**Comment by perry-mitchell** [comment]
Blocked by buttercup/buttercup-core#124 and buttercup/buttercup-core#153, in order.

**Comment by kanallie** [comment]
Hey, 
i just tried it in v0.24.3. It works just fine, if i use "www.google.de" as URL, but it won't work with the value "google.de".
For the test i added the label "URL".

![image](https://user-images.githubusercontent.com/33938386/33185468-15e0f856-d083-11e7-9d96-6deb26812ffa.png)

**Comment by perry-mitchell** [comment]
@kanallie I can confirm this issue, but for me the `www.` prefix doesn't matter. It only works if the prefix is `https://`/`http://`. I'll open a new issue.

**Comment by perry-mitchell** [comment]
I've opened a new issue here: #440
